### PR TITLE
Test for ATS- Node (Supporting #4533)

### PR DESCRIPTION
**Pull request by bmahall**
Supports #4533 

- Adds two new tests supporting the Silent Flow Request in MSAL-node
- The tests check the ATS flow and that the claims hash is passed from the initializeBaseRequest to AcquireToken.

### CODE REVIEW

File: change/@azure-msal-node-a018e8b0-79a2-4e4b-b407-4488edfb0010.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (tnorling) [inline change/@azure-msal-node-a018e8b0-79a2-4e4b-b407-4488edfb0010.json]: Doesn't appear to change the lib in any way
```suggestion
  "type": "none",
```

Comment (bmahall) [reply change/@azure-msal-node-a018e8b0-79a2-4e4b-b407-4488edfb0010.json]: yes, updated that. Thanks!

**Review by hectormmg — APPROVED** [review]
Looks good!

File: change/@azure-msal-node-a018e8b0-79a2-4e4b-b407-4488edfb0010.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Test changes",
```
Comment (jasonnutter) [inline change/@azure-msal-node-a018e8b0-79a2-4e4b-b407-4488edfb0010.json]: Can you please provide a little more detail here? For example:

```suggestion
  "comment": "Tests for SilentFlowClient #4672",
```

Comment (bmahall) [reply change/@azure-msal-node-a018e8b0-79a2-4e4b-b407-4488edfb0010.json]: Sounds good, will keep it in mind!

**Review by jasonnutter — APPROVED** [review]

**Review by sameerag — APPROVED** [review]

**Comment by ghost** [comment]
Reminder: The next release is scheduled for next week and this PR appears to be stale :( 

If changes have been requested please address feedback. 
If this PR is still a work in progress please mark as draft.

**Pull request merged**